contracts/EVMCode.slb

```diff
@@ -31,18 +26,20 @@ library EVMCode {
         }
     }
 
-    function fromArray(RawCode[] memory codes, uint length) internal pure returns (Code memory code) {
-        require(codes.length >= 1, "Empty code array");
+    function fromArray(bytes32[] memory codes, uint length) internal pure returns (Code memory code) {
```

Comment:
Note: I changed it because I got some weird structure in memory using RawCode[]. 
Whatever encoding scheme this was, it wasn't nice and also increased memory usage quite a bit, so I changed it to a simple 'flat' array 🙂

Reply:
I have considered this as well. But I was really like to use js object from client :smile:

Reply:
I was also very surprised once I logged the memory layout and had some strange values between the RawCode array 😅